db/migrate/20240910111203_update_indexes_for_credit_notes_taxes.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class UpdateIndexesForCreditNotesTaxes < ActiveRecord::Migration[7.1]
+  disable_ddl_transaction!
+
+  def change
+    remove_index :credit_notes_taxes, %i[credit_note_id tax_id], unique: true, algorithm: :concurrently
+
+    add_index :credit_notes_taxes, :tax_code, algorithm: :concurrently
+    add_index :credit_notes_taxes, %i[credit_note_id tax_code], unique: true, algorithm: :concurrently
+  end
+end
```

Comment:
there are 1020 CreditNote::AppliedTax on US cluster
and 2330 CreditNote::AppliedTax on EU cluster.

checked uniqueness - everything is uniq